components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -422,7 +437,7 @@ private void updateAppAndRevokeTokensAndAuthzCodes(String consumerKey, Propertie
 
             tokenMgtDAO.updateAppAndRevokeTokensAndAuthzCodes(consumerKey, properties,
                     authorizationCodes.toArray(new String[authorizationCodes.size()]),
-                    accessTokens.toArray(new String[accessTokens.size()]));
+                    accessTokens);
```

Comment:
This line is not properly formatted. You should be able to bring this line to previous line.

Reply:
done with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/472